editor/modes/visual-editor/invalid-block-warning.js

```diff
@@ -54,6 +62,12 @@ export default connect(
 	null,
 	( dispatch, ownProps ) => {
 		return {
+			ignoreInvalid() {
+				const { block } = ownProps;
+				const serializedBlock = serialize( { ...block, isValid: true } );
+				const parsedBlocks = parse( serializedBlock );
+				dispatch( replaceBlock( block.uid, parsedBlocks ) );
```

Comment:
I found a slightly better approach here in 9df0a5d, skipping serialize / parse and instead opting to use the `createBlock` API directly (since we have name and attributes of block already).